docs/kafka/consumer-configuration-kafka/README.adoc

```diff
@@ -74,15 +74,24 @@ NOTE: The Kafka scripts are part of the open source community version of Apache
 endif::[]
 
 //Additional line break to resolve mod docs generation error
+[id="con-consumer-group-list_{context}"]
+== Listing consumer groups
```

Comment:
This structure doesn't quite work for me. This new section ('Listing consumer groups') and the section that follow ('Connecting the Kafka consumer group script') are both geared towards the same user goal - listing the consumer groups for the Kafka instance. So, these should be part of the *same* section, which reflects this user goal.

Therefore, I suggest see something like that shown below. This would introduce a new level 2 section entitled 'Listing consumer groups' with two level 3 sub-sections entitled 'Listing consumer groups using the CLI' and 'Connecting the Kafka consumer group script' within it. I've also suggested some new intro text for each subsection.

[id="con-consumer-group-list_{context}"]
== Listing consumer groups

[role="_abstract"]
As an alternative to using the {product-kafka} web console, you can use the `rhoas` command-line interface (CLI) or the Kafka `kafka-consumer-groups.sh` script to list consumer groups for your Kafka instance. The following subsections describe how to use these methods to list consumer groups 

ifndef::community[]
NOTE: The Kafka scripts are part of the open source community version of Apache Kafka. The scripts are not a part of {product-kafka} and are therefore not supported by Red Hat.
endif::[].

[id="con-consumer-group-list-using-CLI_{context}"]
=== Listing consumer groups using the CLI

[role="_abstract"]
To use the `rhoas` command-line interface (CLI) to list the consumer groups defined for your Kafka instance, enter the following command:

[source,subs="+quotes,+attributes"]
----
rhoas kafka consumer-group list
----

When you enter the preceding command, you should see output like the following example:
<add example output>

[id="con-consumer-group-script_{context}"]
=== Connecting the Kafka consumer group script

[role="_abstract"]
You can list consumer groups using the `kafka-consumer-groups.sh` script. To use this or any other Kafka scripts,
you need to specify the `--bootstrap-server` and `--command-config` flags to connect to your Kafka instance. To list consumer groups, enter the following command:

[source,subs="+quotes,+attributes"]
----
./kafka-consumer-groups.sh --bootstrap-server __<bootstrap_server>__ --command-config __<authentication_properties>__ --list
----

The `<bootstrap_server>` is the bootstrap server endpoint for your Kafka instance.
You copy this information for the Kafka instance in the {product-kafka} web console by selecting the options menu (three vertical dots) and clicking *Connection*.

Kafka scripts require SASL/PLAIN authentication.
Use an `<authentication_properties>` configuration file to configure the connection.
Include the client ID and client secret generated when creating a service account to access your Kafka instance.

.Authentication configuration properties
[source,subs="+quotes"]
----
sasl.mechanism=PLAIN
security.protocol=SASL_SSL

sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
  username="__<client_id>__" \
  password="__<client_secret>__" \;
----

[role="_additional-resources"]
.Additional resources
* {base-url}{getting-started-url-kafka}[_Getting started with {product-long-kafka}_] describes how to create a Kafka instance and service account, and how to set permissions in the instance for a service account.